### PR TITLE
Cache pin

### PR DESCRIPTION
**Pull request by JulianKoch**
Added 4 Buttons to save custom pins with a right click.

### CODE REVIEW

**Comment by alexfupp** [comment]
Added some tasks, see alexfupp@88453696281f095df4b7ef8663213491a79fc54d

**Comment by JulianKoch** [comment]
JulianKoch@17fabc4
-created an own array for pinsaver buttons
-modified setEnableKeySet() for handling the new array
-cleaned code in getCustomPinSaverKey()
-renamed createDefaultButton to createButton
-added and edited javadoc comments

File: de.persosim.driver.connector.ui/src/de/persosim/driver/connector/ui/parts/ReaderPart.java [hunk 1]
```diff
@@ -320,9 +457,21 @@ public void run() {
 				});
         	}
         	break;
+        case KEYS_ALL_PINSAVER:
+        	for(final Button button : keysPinSaver) {
```
Comment (alexfupp) [inline de.persosim.driver.connector.ui/src/de/persosim/driver/connector/ui/parts/ReaderPart.java]: This is the third copy of identical loops, thats definitely too much copied code

**Comment by JulianKoch** [comment]
Reduced the amount of copied code. Ready for next review.

**Pull request merged**